src/main/java/frc/robot/subsystems/SwerveSubsystem.java

```diff
@@ -173,19 +140,174 @@ public void toggleOrientation() {
     isFieldOriented = !isFieldOriented;
     if (!isFieldOriented) {
       // robot oriented drive means we can't hold a field oriented heading
-      endAlignment();
     }
   }
 
-  public void setAlignment(Rotation2d rotation) {
-    alignmentRotation = rotation;
+  public Command applyRequest(Supplier<SwerveRequest> requestSupplier) {
+    return run(() -> this.setControl(requestSupplier.get())).withName("Request Supplier");
   }
 
-  public void endAlignment() {
-    alignmentRotation = null;
+  /**
+   * Request the robot to point in a specified direction any non zero rotation demand will result in
+   * the command canceling.
+   *
+   * @param angleToPoint the angle to point in degrees
+   * @param input the X-Y speeds
+   * @return A command that will point in the specified direction until interupted
+   */
+  public Command pointInDirection(Rotation2d angleToPoint, Supplier<DriveRequest> input) {
+    return pointDrive(() -> angleToPoint.getDegrees(), input).withName("Pointing in direction");
   }
 
-  public Command applyRequest(Supplier<SwerveRequest> requestSupplier) {
-    return run(() -> this.setControl(requestSupplier.get())).withName("Request Supplier");
+  /**
+   * Points a location on the field any non zero rotation demand will result in the command
```

Comment:
Does this mean "Points toward a location on the field"? Also, should be two sentences instead of one.

Reply:
I do appreciate you adding comments, though. :)